cobra/test/test_flux_analysis.py

```diff
@@ -78,15 +68,14 @@ def do_pfba(solver):
     def test_pfba(self, model, solver):
         expression = model.objective.expression
         n_constraints = len(model.solver.constraints)
-        optimize_minimal_flux(model, solver=solver)
-        abs_x = [abs(i) for i in model.solution.x]
+        df = optimize_minimal_flux(model, solver=solver)
+        assert numpy.all([df.columns.values == ['flux', 'objective_value']])
         assert model.solution.status == "optimal"
         if isinstance(model.solution, LegacySolution):
             assert abs(model.solution.f - 0.8739) < 0.001
         else:
-            assert abs(
-                model.solution.fluxes['Biomass_Ecoli_core'] - 0.8739) < 0.001
-        assert abs(sum(abs_x) - 518.4221) < 0.001
+            assert abs(df.flux['Biomass_Ecoli_core'] - 0.8739) < 0.001
```

Comment:
The sum is because we're testing the parsimonious solution, which minimizes the sum of the absolute value. So I think its appropriate. (there might be multiple equally valid parsimonious solutions)